tests/unit/src/test/scala/tests/RenameLspSuite.scala

```diff
@@ -288,7 +288,7 @@ class RenameLspSuite extends BaseRenameLspSuite("rename") {
        |  "" <<::>> user
        |}
        |""".stripMargin,
-    newName = "method:"
+    newName = "+++:"
```

Comment:
That was actually a wrongly named method - `method:` should be wrapped in backticks, which it started doing.